### ISSUE TITLE
Unable to start Tdarr Docker container on Synology (take 2)

### ISSUE THREAD

**Issue by adefaria** [HaveAGitGat/Tdarr#815]
**Describe the bug**

This is a continuation of #722. I was excited that this bug was fixed but it turned out that it's still broken... or I'm doing something wrong.

To recap, I start up Tdarr Server with a script (see #722 for details). In that case, I mentioned restarting the Synology to demonstrate the problem but one can reproduce the problem simply by ssh'ing into the Synology then doing `docker restart tdarr_server`. The container will start but the Web UI will never come up.

I did notice that it does seem to come up very quickly when initially started via my script with the `docker run` command. I noticed a `chown` briefly appear but then goes away. IOW I didn't have to wait 50 minutes for the `chown` to complete. I reported this one in #609.

Note: When pulling the Tdarr Server image I noticed that the version didn't change when using the `latest` tag. I switched to explicitly using `2.00.22` as the tag and verified that the Web UI said 2.00.22. Also, it seems that I need to also specify CHOWN_ON_STARTUP to false. If #609 is truly solved perhaps that should be the default.

**To Reproduce**

1. Pull the 2.00.22 image of the Tdarr Server
2. Use `docker run` to get Tdarr Server up and running
3. Update any Tdarr Nodes to also use 2.00.22
4. Perform a test of transcoding some video of yours to verify that the system is working
5. Ssh into the Synology and issue a `docker restart tdarr_server`
6. You can `docker exec` into the container to watch its progress with `top`.
7. When it settles down attempt to go into the Web UI

**Expected behavior**

I expected the Web UI to come up and function normally

**Please provide the following information:**

[Tdarr_Server_Log.txt](https://github.com/HaveAGitGat/Tdarr/files/12209837/Tdarr_Server_Log.txt)

-Worker error [can be found on the 'Tdarr' tab by pressing the 'i' button on a failed item in the staged file section or in the transcode error section at the bottom]

 - OS: Synology DSM 7.2
 - Version e.g. 2.00.22

**Comment by HaveAGitGat** [comment]
Is this one now resolved for you based on this? https://github.com/HaveAGitGat/Tdarr/issues/817

**Comment by adefaria** [comment]
No. Given my working setup as of last night, this morning I simply ssh'ed into my server and did a docker restart of the container. The container restarted but the web UI never came back up. The container's been up for 9 minutes now and the activity has settled down but the web UI still does not respond.

**Comment by HaveAGitGat** [comment]
I have not had any other Synology users contact me about this in the last 10 months so I can only assume it's something local but feel free to add any findings that come up, thanks.
